test/components/tree-view.spec.tsx

```diff
@@ -41,7 +54,23 @@ describe('<TreeView />', () => {
         simulate.click(elementToSelect);
 
         return waitForDom(() => expect(elementToSelect).to.have.attr('data-selected', 'true'));
```

Comment:
This outcome seems to me like testing implementation... If there's a way to test more behavior that's great.